index.html

```diff
@@ -254,8 +259,42 @@ <h3>LET'S CONNECT</h3>
                         
                     </ul>
                 </div>
+                <!-- start of contact form -->
+                <footer>
+                    <section class="contact-container">
+                        <div class="heading">
+                            <h2>Contact me</h2>
+                        </div>
+                        <div class="content">
+                            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+
+                        </div>
+                        <div class="form-container">
+                            <form
+                            action="https://formspree.io/f/mqkngnbr"
+                            method="POST"
+                          >
+                                <input type="text" id="name" name="username" required 
+                                maxlength="30" value="jemutai"><br><br>
+                          
+                                <input type="Email" id="email" name="email" required
+                                 value="kiplagatjemutai@gmail.com"><br><br>
+                            
+                                <textarea id="subject" name="subject" placeholder="Write your message here.." ></textarea>
```

Comment:
- [ ] I kindly suggest that you add a max length attribute of 500 to your textarea because this will help prevent the submission of a message that's too long.
- [ ] I kindly suggest that you add the required attribute to the textarea input because this will check to make sure the users enters a text in the textarea